ui/src/app/workflows/components/workflow-details/workflow-details.tsx

```diff
@@ -43,6 +43,7 @@ import {SuspendInputs} from './suspend-inputs';
 import {WorkflowResourcePanel} from './workflow-resource-panel';
 
 import './workflow-details.scss';
+import {useTranslation} from 'react-i18next';
```

Comment:
same as above, this should be at the top of the file with the other external imports